decidim-core/app/controllers/decidim/amendments_controller.rb

```diff
@@ -219,10 +210,6 @@ def ensure_is_draft_from_user
       raise ActionController::RoutingError, "Not Found" unless amendment.draft? && amender == current_user
     end
 
-    def similar_emendations
-      @similar_emendations ||= Decidim::SimilarEmendations.for(amendment)
```

Comment:
Can you remove this file 🙏🏽 ?  ./decidim-core/app/queries/decidim/similar_emendations.rb 

It isn't  neeeded anymore